logos/tests/fixtures/scheduling/README.md

```diff
@@ -0,0 +1,23 @@
+# Scheduling Workload CSVs
+
+Each workload lives in this directory and must be a UTF-8 CSV with the following header row:
+
+```
+request_id,arrival_offset,prompt,mode[,body_json][,body_template]
+```
+
+Field details:
+
+- `request_id` *(optional)*: unique identifier for the row. If omitted, IDs are auto-generated in row order.
+- `arrival_offset` *(required)*: float seconds (relative to the start of the replay) when this request should be issued.
+- `prompt` *(required)*: user-facing text used in the default payload or template substitution.
```

Comment:
Why is prompt always required? Couldn't I simply use either: (prompt + body_template) OR body_json only?